railties/lib/rails/generators/rails/devcontainer/templates/devcontainer/compose.yaml.tt

```diff
@@ -39,8 +40,10 @@ services:
 
 <%- end -%>
   <%= devcontainer_db_service_yaml(indentation: 4) %>
-<%- if !volumes.empty? -%>
+
 volumes:
+  rbenv-data:
+<%- if !volumes.empty? -%>
 <%- volumes.each do |volume| -%>
   <%= volume %>:
 <%- end -%>
```

Comment:
You don't need the `if` anymore, it was there to avoid an empty `volumes:`

```suggestion
<%- volumes.each do |volume| -%>
  <%= volume %>:
```

Reply:
Thanks, Rune. 💯